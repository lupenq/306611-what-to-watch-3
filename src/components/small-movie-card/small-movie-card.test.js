import SmallMovieCard from './small-movie-card';

it(`Render SmallMovieCard`, () => {
  const IMG_URL = `https://api.adorable.io/avatars/128`;

  const films = {
    name: `Интерстеллар`,
    picture: `${IMG_URL}`,
    id: 1,
    play: true,
    preview: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  };

  const tree = renderer
  .create(
      <SmallMovieCard
        name={films.name}
        picture={films.picture}
        id={films.id}
        key={films.name}
        play={films.play}
        preview={films.preview}
        onMovieCardHover={() => {}}
        onMovieCardUnhover={() => {}}
        onMovieCardTitleClick={() => {}}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
