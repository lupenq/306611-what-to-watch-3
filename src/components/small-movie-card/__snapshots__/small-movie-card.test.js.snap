// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render SmallMovieCard 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onClick={[Function]}
  onMouseOut={[Function]}
  onMouseOver={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <video
      autoPlay={true}
      muted={true}
      onClick={[Function]}
      poster="https://api.adorable.io/avatars/128"
      width="100%"
    >
      <source
        src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
      />
    </video>
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
      onClick={[Function]}
    >
      Интерстеллар
    </a>
  </h3>
</article>
`;
